.github/workflows/pr-build.yml

```diff
@@ -67,6 +67,20 @@ jobs:
         with:
           arguments: build integrationTests --stacktrace -PenableCoverage=true -PlocalDocker=true
 
+      - name: Get current version
```

Comment:
Are we still planning to pull this pr-build workflow for cp-utility out in a separate workflow file?

Reply:
I'm debating myself if there is much benefit in separating. I think if we change the tool, we will also want to build the image, which will imply in having to build the jar, therefore we would be duplicating much of what is done in this workflow. 

Do you think it is worth separating? what are the pros? The only thing I can think of is that we would not need to run integration tests.

Reply:
ok. thanks. I agree this would duplicate the building jar process. Should we add these steps to main-build.yml as well?

Reply:
I will add to main build.